tests/Doctrine/Tests/DBAL/Functional/Schema/SqliteSchemaManagerTest.php

```diff
@@ -173,7 +173,7 @@ public function testNonDefaultPKOrder() : void
 CREATE TABLE non_default_pk_order (
     id INTEGER,
     other_id INTEGER,
-    PRIMARY KEY(other_id, id)
+    PRIMARY KEY (other_id, id)
```

Comment:
Please revert the changes like this. It's fine if we want to adjust the formatting (it will need to be justified) but I'd rather do this in a separate PR to not mix two unrelated changes.

Reply:
The problem is that currently the codebase is not consistent. Sometimes is `PRIMARY KEY(id)` and sometimes `PRIMARY KEY (id)`. In this PR i have unified the generation of the primary key definition, thus only one way will work

Reply:
The unification is fine but I'd rather do it in a separate PR.

Reply:
I can try to split it, but then that PR needs to be merged before this

Reply:
How about just a separate commit? We can review commit by commit.

Reply:
For me that would help a lot

Reply:
A separate commit is fine too. Also, I'd suggest to not have more commits than needed to help the reviewer (i.e. 2 in this case). Having intermediate commits that fail the build sometimes results in horrible `git bisect` experience. If those in-between commits are not valuable from the reviewer's perspective, I'd squash them.